src/error/syntaxError.js

```diff
@@ -22,9 +22,11 @@ export function syntaxError(
   description: string
 ): GraphQLError {
   const location = getLocation(source, position);
+  const line = location.line + source.location.line - 1;
```

Comment:
We should probably also handle the column offset too, but only on the first line. eg. if my source looks like this (ie. starting line 2, starting column 24):

```
  12345678901234567890123456789
1:
2:     graphql` ... X ...
3:  Y ...
4:`
```

If there is an error at "X", we should report that as being on line 2, column 29, and if there is an error at "Y" then we should report it as being on line 3, column 3. Does that make sense?